### ISSUE TITLE
Travis is blowing up on iojs build

### ISSUE THREAD

**Issue by ilyavolodin** [eslint/eslint#2117]
With the following error:

```
path.js:8
    throw new TypeError('Path must be a string. Received ' +
          ^
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.dirname (path.js:539:3)
    at Deps.lookupPackage (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/module-deps/index.js:433:29)
    at Deps._transform (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/module-deps/index.js:85:10)
    at Deps.Transform._read (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at Deps.Transform._write (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at Labeled.Pipeline._write (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/index.js:72:22)
    at doWrite (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:279:12)
    at writeOrBuffer (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:266:5)
    at Labeled.Writable.write (/home/travis/build/eslint/eslint/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:211:11)
Error: Cannot find module 'espree'
```
I assume it's trying to verify licensing for espree and for some reason it can't.

**Comment by gyandeeps** [comment]
It's should not be a issue any more as they fixed in iojs 1.6.1
Hers is the fixed commit https://github.com/iojs/io.js/pull/1216
Try building your PR again

**Comment by nzakas** [comment]
Looks like this is fine now.
